pkg/buildpipeline/pipeline.go

```diff
@@ -711,16 +743,23 @@ func (j *Jenkinsfile) GenerateCRDs(pipelineIdentifier string, buildIdentifier st
 			Name:      fmt.Sprintf("%s-build-%s-%s", pipelineIdentifier, buildIdentifier, suffix),
 		},
 		Spec: pipelinev1alpha1.PipelineSpec{
-			Resources: []pipelinev1alpha1.PipelineDeclaredResource{{
-				Name: "common-workspace",
-				Type: pipelinev1alpha1.PipelineResourceTypeGit,
-			}},
+			Resources: []pipelinev1alpha1.PipelineDeclaredResource{
+				{
+					Name: "common-workspace",
+					Type: pipelinev1alpha1.PipelineResourceTypeGit,
+				},
+				{
+					// TODO: Switch from this kind of hackish approach to non-resource-based dependencies once they land.
```

Comment:
I feel like we might want to use a different resource type here than `git` - something with less mechanical baggage attached?

Reply:
Looks like the choices are `git`, `image`,`cluster`, or `storage`, so I guess switch to `image`?

Reply:
Yeah, I think `image` is basically a no-op right now, so that'd work.

Reply:
Done in d6f0c8f.